tests/core/recipes/vasp_recipes/mocked/test_vasp_recipes.py

```diff
@@ -849,3 +850,12 @@ def test_mp_relax_flow_custom(tmp_path, patch_nonmetallic_taskdoc):
             job_params={"mp_metagga_relax_job": {"nsw": 0}},
         )
         assert output["relax2"]["parameters"]["nsw"] == 0
+
+
+def test_freq_job():
+    atoms = molecule("H2")
+    atoms.pbc = True
+    atoms.center(vacuum=1)
+    output = freq_job(atoms, kpts=(1, 1, 1))
+    assert output["parameters"]["ediff"] == 1e-07
+    assert len(output["results"]["vib_freqs_raw"]) == 3 * len(atoms)
```

Comment:
Could we also test the "actual" number of vibrational modes (3N-5 for linear)? Just to make sure that's being done correctly...

Reply:
It seems that the symmetry isn't being read for the 'ideal_gas' case; so the "vib_freqs" doesn't differ from "vib_freq_raw"